tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/SearchConfigIndexingSpec.scala

```diff
@@ -871,6 +876,188 @@ class SearchConfigIndexingSpec extends BaseIntegrationSpec {
         aggregationIn(json) should contain(preSynapticBrainRegionAgg)
       }
     }
+
+    "have the correct morphology mean length" in {
+      val query    = queryField(neuronMorphologyId, "morphologyFeature")
+      val expected =
+        json"""[
+                {
+                 "value" : 1131.963147431612,
+                 "unit" : "μm",
+                 "label" : "Total Length",
+                 "statistic" : "mean",
+                 "compartment" : "NeuronMorphology",
```

Comment:
The comma on this line will cause the test to fail

Reply:
comma removed